participant/Dockerfile

```diff
@@ -2,58 +2,55 @@
 # Base node image with package updates and dependencies
 FROM node:18-slim as base
 # Install openssl for Prisma
-WORKDIR /srv
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
   openssl
+# Set working directory and node user
+WORKDIR /home/node/remix
 
 # -------- Image: deps -------- #
 # Install all node_modules, including dev dependencies
 FROM base as deps
-COPY package.json package-lock.json ./
+COPY --chown=node:node package.json package-lock.json ./
 RUN npm install --production=false
 
 # -------- Image: dev -------- #
 # Provide a development image
 FROM base as dev
-COPY --from=deps /srv/node_modules /srv/node_modules
-COPY prisma .
+COPY --from=deps /home/node/remix/node_modules ./node_modules
+COPY --chown=node:node prisma .
 RUN npx prisma generate
-COPY . .
+COPY --chown=node:node . .
 CMD ["npm", "run", "dev:docker"]
 
 # -------- Image: build -------- #
 # Build the app
 FROM base as build
-COPY --from=deps /srv/node_modules /srv/node_modules
-COPY prisma .
+COPY --from=deps /home/node/remix/node_modules ./node_modules
+COPY --chown=node:node prisma .
 RUN npx prisma generate
-COPY . .
-RUN npm run build
+COPY --chown=node:node . .
 RUN npm run css
-
-# -------- Image: test -------- #
-# Provide a test image
-FROM build as test
-ENV NODE_ENV test
-CMD ["npm", "test"]
+RUN npm run build
 
 # -------- Image: production-deps -------- #
 # Setup production node_modules
 FROM base as production-deps
 ENV NODE_ENV production
-COPY --from=deps /srv/node_modules /srv/node_modules
-COPY package.json package-lock.json ./
+COPY --from=deps /home/node/remix/node_modules ./node_modules
+COPY --chown=node:node package.json package-lock.json ./
```

Comment:
not clear on why we need the permissions changes now if they're being created as the default user for the image 🤔

Reply:
I'll try without and check. I wasn't sure if it was bringing over the owner:group from the host

Reply:
I tried without it and a few other configurations and you end up with files owned by root:root instead. I'm going to leave it as is.